template.yaml

```
@@ -68,7 +68,7 @@ Conditions:
 Mappings:
   Solution:
     Constants:
-      Version: '1.0'
+      Version: '1.0.1'
```

```suggestion
      Version: '1.1'
```
I think with most new projects we tend to simplify by just major/minor - like safe to upgrade vs breaking.